### PR TITLE
Final hack 1

### PR DESCRIPTION
**Pull request by rubenpulido1**
Added in new comments and put back in create your own function

### CODE REVIEW

File: js/hacks/hack1_NIKO_HACKED_YOU/hack1_niko_wuz_here_lol.js [hunk 1]
```diff
@@ -1,29 +1,48 @@
...
 }
+
+//Here's a challenge!
+function nameYourFunction(message) {
+    /*
+    Functions can also take arguments which can be numbers, strings, etc. In this case
+    the argument is named "message" but it can be named anything.
+    */
+
+    /*
```
Comment (yuxigui) [inline js/hacks/hack1_NIKO_HACKED_YOU/hack1_niko_wuz_here_lol.js]: Can we merge the two block comments into 1 block comment to look nicer?

File: js/hacks/hack1_NIKO_HACKED_YOU/hack1_niko_wuz_here_lol.js [hunk 2]
```diff
@@ -1,29 +1,48 @@
+//CAREFULLY READ COMMENTS, All text in gray is a comment.
```
Comment (yuxigui) [inline js/hacks/hack1_NIKO_HACKED_YOU/hack1_niko_wuz_here_lol.js]: also combine the comments here

File: js/hacks/hack1_NIKO_HACKED_YOU/hack1_niko_wuz_here_lol.js [hunk 3]
```diff
@@ -1,29 +1,48 @@
...
     //BONUS: Can you print another message below the first?
     return "Niko wuz here";
 }
 
 
-/* The following code below are functions, which are a block of code designed to perform
-   a particular tasks. A function is executed when it is called 
+/*
+The following code below are functions, which are a block of code designed to perform
+a particular tasks. A function is executed when it is called
```
Comment (yuxigui) [inline js/hacks/hack1_NIKO_HACKED_YOU/hack1_niko_wuz_here_lol.js]: period after called

File: js/hacks/hack1_NIKO_HACKED_YOU/hack1_niko_wuz_here_lol.js [hunk 4]
```diff
@@ -1,29 +1,48 @@
...
 */
 
-
 function eliminateVirus() {
     // When this function is called it will return the string below
     // How to use: write eliminateVirus after the return statement in the main function
     return "Virus has been eliminated";
 }
+
+//Here's a challenge!
```
Comment (yuxigui) [inline js/hacks/hack1_NIKO_HACKED_YOU/hack1_niko_wuz_here_lol.js]: Add clarity we mentioned in meeting!

File: js/hacks/hack1_NIKO_HACKED_YOU/hack1_niko_wuz_here_lol.js (hunk 2 above)
Comment (yuxigui) [inline js/hacks/hack1_NIKO_HACKED_YOU/hack1_niko_wuz_here_lol.js]: period after comments

**Review by yuxigui — COMMENTED** [review]
make the comments look more organized would be nice!

**Review by yuxigui — APPROVED** [review]

**Review by develop4learners — APPROVED** [review]
It ran for me when I used nameYourFunction, and saw my output from nameYourFunction in the console. Really nice work! 

I also added 1commit that changes main() to hack1_main() and updates the places its called in the check file.

**Review by MeshanKhosla — APPROVED** [review]

**Pull request merged**